### ISSUE TITLE
Incorrect text in log when using `/bl 1`

### ISSUE THREAD

**Issue by Herschel** [onimitch/ffxi-balloon-ashitav4#4]
When using `/bl 1`, certain non-NPC messages are incorrectly truncated, causing extraneous characters to appear in the chat log.

Example:
![Hersch_2024 01 25_195225](https://github.com/onimitch/ffxi-balloon-ashitav4/assets/36278/7a726592-9446-4a09-92d2-5ce7f489690d)

Message data:
```
"\x1e\x01A home point can be set as a spot for you to return to Vana'diel when you have been knocked out. You can also use a home point to teleport to other home points."
```

**Comment by onimitch** [comment]
I think I might already have this fixed but I'll double check with your test case, thanks.

**Comment by onimitch** [comment]
Fixed in [Version 4.0.4](https://github.com/onimitch/ffxi-balloon-ashitav4/releases/tag/4.0.4)
